lib/MailQueueHandler.php

```diff
@@ -382,7 +382,7 @@ protected function sendEmailToUser($userName, $email, $lang, $timezone, $maxTime
 			'activityEvents' => $activityEvents,
 			'skippedCount' => $skippedCount,
 		]);
-		$template->setSubject($l->t('Activity notification for %s', $this->getSenderData('name')));
+		$template->setSubject($l->t('Activity on %s', $this->getSenderData('name')));
```

Comment:
So it would be something like "Activity on Louis"?
I am not sure of the context, but it sounds weird. Maybe "Activity from" or "Activity for"?

Reply:
Oh, ok then, that makes sense. @tobiasKaminsky can you confirm?